meshroom/core/plugins.py

```diff
@@ -126,6 +126,14 @@ def getEnvDict(self) -> dict:
         env["LD_LIBRARY_PATH"] = f"{os.pathsep.join(self.libPaths)}{os.pathsep}{os.getenv('LD_LIBRARY_PATH', '')}"
         env["PATH"] = f"{os.pathsep.join(self.binPaths)}{os.pathsep}{os.getenv('PATH', '')}"
 
+        for k, val in self._configEnv.items():
+            # Preserve user-defined environment variables:
+            # manually set environment variable values take precedence over config file defaults.
+            if k in env:
+                continue
+
+            env[k] = val
+
```

Comment:
Is it still needed to modify the process env, as we now evaluate the value using the config in the evalValue?

Reply:
Yes, because some environment variables are not paths to objects but need to be set at runtime. Example here: https://github.com/meshroomHub/mrSegmentation/blob/main/meshroom/config.json#L28